src/cozmo/robot.py

```diff
@@ -1106,6 +1132,43 @@ def start_behavior(self, behavior_type):
         await asyncio.sleep(active_time, loop=self._loop)
         b.stop()
 
+    def _stop_behavior(self):
+        # Internal helper method called from Behavior.stop etc.
+        msg = _clad_to_engine_iface.ExecuteBehaviorByExecutableType(
+                behaviorType=_clad_to_engine_cozmo.ExecutableBehaviorType.NoneBehavior)
+        self.conn.send_msg(msg)
+        self._is_behavior_running = False
+
+    def start_freeplay_behaviors(self):
+        '''Start running freeplay behaviors on Cozmo
+
+        Puts Cozmo into a freeplay mode where he autonomously drives around
+        and does stuff based on his mood and environment.
+
+        You shouldn't attempt to drive Cozmo during this, as it will clash
+        with whatever the behaviors are attempting to do internally.
```

Comment:
"with whatever the behaviors are attempting to do internally"

Maybe this could be reworded, something like "with whatever behaviors the robot is attempting"?